js/components/form/tag-completer.vue

```diff
@@ -11,11 +12,16 @@ export default {
     selectize: {
         valueField: 'text',
         plugins: ['remove_button'],
-        create: function(input) {
+        create(input) {
             return {
                 value: input,
                 text: input
             }
+        },
+        createFilter(value) {
```

Comment:
Comparison operators cannot be chained in JS.
Each part return a `boolean`. And you don't want to compare a `boolean` with the rest of your expression.